docs/gallery/domain/plot_behavior.py

```diff
@@ -0,0 +1,415 @@
+
+session_start_time = datetime(2018, 4, 25, 2, 30, 3, tzinfo=tz.gettz("US/Pacific"))
+
+nwbfile = NWBFile(
+    session_description="Mouse exploring an open field",  # required
+    identifier="Mouse5_Day3",  # required
+    session_start_time=session_start_time,  # required
+    session_id="session_1234",  # optional
+    experimenter="My Name",  # optional
+    lab="My Lab Name",  # optional
+    institution="University of My Institution",  # optional
+    related_publications="DOI:10.1016/j.neuron.2016.12.011",  # optional
+)
+
+nwbfile
+
+####################
+# Position
+# ------------
+#
+# Spatial Series
+# ^^^^^^^^^^^^^^^
+#
+# :py:class:`~pynwb.behavior.SpatialSeries` is a subclass of :py:class:`~pynwb.base.TimeSeries`
+# that represents the spatial direction, e.g., of gaze or travel, or position of an animal over time.
+#
+# .. seealso::
+#    You can learn more about the :py:class:`~pynwb.behavior.SpatialSeries` data type and
+#    :py:class:`~pynwb.behavior.Position` interface in the :ref:`basic_spatialseries` tutorial.
+#
+# Create data that corresponds to x, y position over time.
+
+position_data = np.array([np.linspace(0, 10, 50), np.linspace(0, 8, 50)]).T
+
+position_data.shape
+
+####################
+# In :py:class:`~pynwb.behavior.SpatialSeries` ``data``, the first dimension is always time (in seconds),
+# the second dimension represents the x, y position (in meters).
+#
+# .. note::
+#    :py:class:`~pynwb.behavior.SpatialSeries` data should be stored as one continuous stream,
+#    as it is acquired, not by trial as is often reshaped for analysis.
+#    Data can be trial-aligned on-the-fly using the trials table. See the :ref:`basic_trials` tutorial
+#    for further information.
+#
+# For position data ``reference_frame`` indicates the zero-position, e.g.
+# the 0,0 point might be the bottom-left corner of an enclosure, as viewed from the tracking camera.
+
+timestamps = np.linspace(0, 50) / 200
+
+position_spatial_series = SpatialSeries(
+    name="SpatialSeries",
+    description="(x,y) position in open field",
+    data=position_data,
+    timestamps=timestamps,
+    reference_frame="(0,0) is bottom left corner",
+)
+
+position_spatial_series
+
+####################
+#
+# .. seealso::
+#    You can learn more about best practices that can be applied to
+#    :py:class:`~pynwb.behavior.SpatialSeries` `here <https://www.nwb.org/best-practices/#timeseries>`_.
+#
+# To help data analysis and visualization tools know that this :py:class:`~pynwb.behavior.SpatialSeries` object
+# represents the position of the subject, store the :py:class:`~pynwb.behavior.SpatialSeries` object inside
+# of a :py:class:`~pynwb.behavior.Position` object, which can hold one or more :py:class:`~pynwb.behavior.SpatialSeries` objects.
+
+position = Position(spatial_series=position_spatial_series)
+
+
+####################
+# Create a Behavior Processing Module
+# --------------------------
```

Comment:
Please fix the underlines "-----" on the headings to match the length of the title. If the underlines are shorter than the heading text this results in warnings during the built.